README.md

```diff
@@ -98,19 +98,17 @@ but usually we will act ASAP anyway.
 
 after you have been approved..
 
-check if your pubkey file is in the current master of https://github.com/krebs/retiolum
-look into your pubkey file and copy your ipv6 address which got autogenerated.
+To learn your IP address, run `nix eval --impure --expr '((import ./lib).krebs.genipv6 "retiolum" "<your_username>" { hostName = "<your_hostname>"; }).address'` in the stockholm repo.
```

Comment:
This is actually not necessarily true. `genipv6` is an optional convenience function, even though everybody uses it (I think.)  The only structure that's mandatory is `42:${network-id}:${hash username}::/48`, the other bits are user-specified.